gradle/maven.gradle

```diff
@@ -22,13 +22,13 @@ subprojects {
   publishing {
     publications {
       mavenJava(MavenPublication) {
-        from components.java
-        if(project.name == 'samza-shell') {
-          artifact shellTarGz {
-            classifier "dist"
```

Comment:
Will there be an impact to the artifact name if `classifier "dist"` is removed here?

Reply:
The artifact produced by task `shellTarGz` already specifies the `dist` classifier: https://github.com/apache/samza/blob/master/build.gradle#L552

Just in case, I'll build again and verify locally.

Reply:
Verification details.

Before (master branch at 851fc48):
```
$ ./gradlew clean :samza-shell:assemble
$ ls -al samza-shell/build/distributions/samza-shell-1.4.0-SNAPSHOT-dist.tgz 
-rw-r--r-- 1 kmoore eng 96811 Mar  3 09:48 samza-shell/build/distributions/samza-shell-1.4.0-SNAPSHOT-dist.tgz
$ shasum samza-shell/build/distributions/samza-shell-1.4.0-SNAPSHOT-dist.tgz
40acd2debccda136a7dee1a2647eb29ddb88df04  samza-shell/build/distributions/samza-shell-1.4.0-SNAPSHOT-dist.tgz
```
After (PR branch):
```
$ ./gradlew clean :samza-shell:assemble
$ ls -al samza-shell/build/distributions/samza-shell-1.4.0-SNAPSHOT-dist.tgz 
-rw-r--r-- 1 kmoore eng 96811 Mar  3 09:50 samza-shell/build/distributions/samza-shell-1.4.0-SNAPSHOT-dist.tgz
$ shasum samza-shell/build/distributions/samza-shell-1.4.0-SNAPSHOT-dist.tgz
40acd2debccda136a7dee1a2647eb29ddb88df04  samza-shell/build/distributions/samza-shell-1.4.0-SNAPSHOT-dist.tgz
```
Checksums are identical and filenames match, including classifier.